src/Package/Impl/Packages/R/RPackage.cs

```diff
@@ -140,7 +140,10 @@ internal class RPackage : BasePackage<RLanguageService>, IRPackage {
 
             LogCleanup.Cancel();
             ProjectIconProvider.Close();
-            CsvAppFileIO.Close(Services.FileSystem());
+
+            if (Services?.FileSystem() != null) {
```

Comment:
This was one-off crash when package failed to load and got disposed before services were actually populated. Sure, I can store it.